internal/scan/scan_test.go

```diff
@@ -28,6 +30,10 @@ func TestRunLocalScan(t *testing.T) {
 
 			var cfg types.Config
 			cfg.NewDefaultConfig() // Assuming a method to set up default config
+			_, err := toml.DecodeFile("../../config.toml", &cfg)
+			if err != nil {
+				t.Errorf("Failed to parse configuration file: %s", err)
+			}
```

Comment:
Why is this added here? We do test config files elsewhere (see `dist/releases/config_test.go`).

Reply:
This is a byproduct of not having a default value for certified distributions.

It appears the configuration we're using in the test, which is initialized on line 32, is using a set of reasonable defaults:

  https://github.com/openshift/check-payload/blob/main/internal/types/types.go#L137-L165

But, the certified distributions are empty.

Should we add in a default value instead?

Reply:
> Should we add in a default value instead?

Maybe.

Also, as I mentioned [here](https://github.com/openshift/check-payload/pull/154#discussion_r1480835381), the NewDefaultConfig should be in a `_test.go` file since we're only using it for testing.

Maybe we can even remove `NewDefaultConfig` entirely (as the name is misleading, and it is only used by a single test case), and have some config right there in `TestRunLocalScan`.

Opened https://github.com/openshift/check-payload/pull/191

Reply:
Rebased on top of your patch after it merged and remove the modifications to `config.toml` by adding it to the testing configuration directly.